components/uploader.py

```diff
@@ -0,0 +1,57 @@
+import dash_html_components as html
+import dash_core_components as dcc
+import dash_table
+from utils import dataparser
+
+
+def get_div():
+    """
+    Return the uploader component
+    """
+    return html.Div([
+        dcc.Upload(
+            id='upload-data',
+            children=html.Div([
+                'Drag and Drop or ',
+                html.A('Select Files')
+            ]),
+            style={
+                'width': '95%',
+                'height': '60px',
+                'lineHeight': '60px',
+                'borderWidth': '1px',
+                'borderStyle': 'dashed',
+                'borderRadius': '5px',
+                'textAlign': 'center',
+                'margin': '2.5%'
+            }
+        ),
+        html.Div(id='output-data-upload'),
+    ])
+
+
+def update_output(contents, filename):
+    """
+    Update the data table on the page, or display error
+    message if parsing fails.
+    """
```

Comment:
Not necessary, but we might want to consider having argument documentation as well. At least to me it's not obvious what these arguments are doing before I read the code. What do you think? See [PEP 257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings) for the official format.

Reply:
Added the documentation about arguments to all functions where any are used